azure/template.json

```diff
@@ -119,9 +124,6 @@
                     },
                     "attachedService": {
                         "value": "[variables('appServiceName')]"
-                    },
-                    "logAnalyticsWorkspaceId": {

```

Comment:
This shouldn't be removed

Reply:
Should be logAnalyticsWorkspaceId and the value should be what is was set as before it was removed